GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -228,6 +229,12 @@ public FormCommit(GitUICommands commands, CommitKind commitKind = CommitKind.Nor
             {
                 gpgSignCommitToolStripComboBox.SelectedIndex = 0;
             }
+
+            ((ToolStripDropDownMenu)commitTemplatesToolStripMenuItem.DropDown).ShowImageMargin = false;
+            ((ToolStripDropDownMenu)commitTemplatesToolStripMenuItem.DropDown).ShowCheckMargin = false;
+
+            ((ToolStripDropDownMenu)commitMessageToolStripMenuItem.DropDown).ShowImageMargin = false;
+            ((ToolStripDropDownMenu)commitMessageToolStripMenuItem.DropDown).ShowCheckMargin = false;
```

Comment:
these double casts look uncharacteristic for you, Drew :)

Reply:
Haha, I don't know what came over me :)